src/node_task_runner.cc

```diff
@@ -153,12 +153,19 @@ std::string EscapeShell(const std::string_view input) {
     return std::string(input);
   }
 
+#ifdef _WIN32
+  // Wrap the result in double quotes
+  // Passing "--help "sdfsd"" to a batch script
+  // will be considered as a single argument made
+  // of --help "sdfsd". However, there are corner cases.
+  std::string escaped = "\"" + escaped + "\"";
```

Comment:
We need to also escape `"` characters with `\"` and update `cctest/node_task_runner.cc` file

Reply:
> We need to also escape " characters with \" 

Do you have a reference that says so? I don't think it works this way:

https://stackoverflow.com/questions/7760545/escape-double-quotes-in-parameter

Reply:
That's nice. TIL. Can you also update `cctest/node_task_runner.cc`

Reply:
I did update it.

Reply:
@anonrig 

So `"Tim says, ""Hi!"""`  and `"Tim says, "Hi!""`  become `Tim says, "Hi!"`. There is a fairly complex algorithm that spans quite a few paragraphs.